cmd/vclusterctl/cmd/platform/login.go

```diff
@@ -133,6 +133,7 @@ func (cmd *LoginCmd) Run(ctx context.Context, args []string) error {
 	if err != nil {
 		return err
 	}
+	cmd.Log.Infof("Login credentials are stored in %q", cmd.Config)
```

Comment:
what do you think about combining these messages? Just thinking that it might get missed. Example:
```suggestion
	cmd.Log.Donef(product.Replace("Successfully logged into Loft instance %s, credentials are stored in %q"), ansi.Color(url, "white+b"), cmd.Config)
```